ang/civiawards/awards/AwardsNew.html

```diff
@@ -32,8 +32,8 @@ <h1 crm-page-title>{{title}}</h1>
           </label>
           <div class="col-sm-5">
             <input class="form-control"
-              ng-model="selectedActivityType"
-              crm-ui-select="{
+                   ng-model="selectedActivityType"
```

Comment:
This file seems to only have whit-space changes, is this intentional?

Reply:
I merged this in from the `RSE-305-create-an-award` branch. It didn't have a conflict so that must be how it is in original branch.